### PR TITLE
Native: privacyLink is now converted to ortb.privacy 

### PR DESCRIPTION
**Pull request by musikele**
## Type of change

- [X] Bugfix

## Description of change
In legacy native, users could request `privacyLink`. Current prebid ORTB implementation was not considering this value. We decided that, if `privacyLink: { required: true }`, we set `ortb.privacy = 1`. 

## Other information
Discussion: https://github.com/prebid/Prebid.js/issues/10249

### CODE REVIEW

File: src/native.js [hunk 1]
```diff
@@ -480,6 +480,11 @@ export function toOrtbNativeRequest(legacyNativeAssets) {
       continue;
     }
 
+    if (key === 'privacyLink') {
+      ortb.privacy = 1;
```
Comment (dgirardi) [inline src/native.js:484]: If I say `privacyLink: {required: false}` should this still be set to 1?

**Comment by dgirardi** [comment]
I believe this also needs a change in `legacyPropertiesToOrtbNative`, to address @jsnellbaker's point.

File: src/native.js (hunk 1 above)
Comment (dgirardi) [reply src/native.js:484]: in ORTB the flag signals support and not whether the link is required, so maybe this is OK - but we should set `required: false` in the inverse logic?

Comment (musikele) [reply src/native.js:484]: My takes on this

> If I say privacyLink: {required: false} should this still be set to 1? 
I'd say the presence of the attribute means that you'd like to set privacy to 1. 

> in ORTB the flag signals support and not whether the link is required, so maybe this is OK - but we should set required: false in the inverse logic? 
Agree, will fix

File: src/native.js [hunk 2]
```diff
@@ -696,8 +704,11 @@ export function legacyPropertiesToOrtbNative(legacyNative) {
         // in general, native trackers seem to be neglected and/or broken
         response.jstracker = Array.isArray(value) ? value.join('') : value;
         break;
+      case 'privacyLink':
+        response.privacy = 1;
```
[the remark this answers is not in the document]
Comment (musikele) [reply src/native.js:708]: Yeah you are definitely right ----

**Review by dgirardi — APPROVED** [review]

**Pull request merged**